### PR TITLE
HTML first draft

### PR DESCRIPTION
**Pull request by jdselby24**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <script type="text/javascript" src="" async></script>
+    <link rel="stylesheet" type="text/css" href="">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
```
Comment (sudokufan) [inline index.html]: think we should save this locally to the repo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <script type="text/javascript" src="" async></script>
+    <link rel="stylesheet" type="text/css" href="">
```
Comment (sudokufan) [inline index.html]: should fill in styles.css?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+</head>
+
+<body>
+<div class="container">
+    <div class="menu">
+        <button class="instructionsButton"></button>
+        <button class="newGame"></button>
```
Comment (jdselby24) [inline index.html]: Write 'New Game' between button tags

```suggestion
        <button class="newGame">New Game</button>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+    <script type="text/javascript" src="" async></script>
+    <link rel="stylesheet" type="text/css" href="">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+</head>
+
+<body>
+<div class="container">
+    <div class="menu">
+        <button class="instructionsButton"></button>
```
Comment (jdselby24) [inline index.html]: Write 'Instructions' between button tags
```suggestion
        <button class="instructionsButton">Instructions</button>
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <button class="newGame"></button>
+        <div class="score"></div>
+        <div class="timer"></div>
+    </div>
+
+    <div class="instructionsContainer"></div>
```
Comment (jdselby24) [inline index.html]: Insert instruction text in p tags inside this div

File: index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        <button class="instructionsButton"></button>
+        <button class="newGame">New Game</button>
+        <div class="score"></div>
+        <div class="timer"></div>
+    </div>
+
+    <div class="instructionsContainer"></div>
+
+    <div class="cardContainer">
+        <div class="card" id="c0"></div>
```
Comment (jdselby24) [inline index.html]: Looks good!

**Pull request merged**